README.md

```diff
@@ -15,25 +15,68 @@ For more info click [here][switchbot-api-repo]
 
 
 ## Requirements
-- HACS
-    - PyScript Integration
+- HACS ([doc](https://hacs.xyz/docs/setup/prerequisites))
+    - PyScript Integration ([doc](https://hacs-pyscript.readthedocs.io/en/latest/installation.html)) with `import-all` option activated
+      
 
 ## Installation
-1. Simply clone this repo and copy the pyscript folder in your home assisant config folder 
-2. Include (or copy) [`pyscript/config.yaml`](./pyscript/config.yaml) in configuration.yaml under -> pyscript:
-3. In [`pyscript/config.yaml`](./pyscript/config.yaml), set and search the following parameters (secret suggested):
-    - Token (`token:`) from `Developer Option` in the SwitchBot App
-    - Secret Key (`secret:`) from `Developer Option` in the SwitchBot App
-    - Random Valude (`nonce:`) I suggest using an uuid generaotr, but any alphanumeric string is fine.
+### Procedure
+1. **Clone this repository in the `deps/` directory**
+   ```sh
+   cd deps/
+   git clone https://github.com/SiriosDev/SwitchBot-API-Script-Caller.git
+   cd ..
+   ```
+3. **Include [`pyscript/config.yaml`](./pyscript/config.yaml) in `configuration.yaml` under the `pyscript` section**
+   ```yaml
+   # configuration.yaml
+   # (...)
+   pyscript: !include deps/SwitchBot-API-Script-Caller/pyscript/config.yaml
+   # (...)
+   ```
+5. **Set the authentication secrets in `secrets.yaml` homeassistant file**
+    - Random Value (`switchbot_nonc`) (I suggest using an UUID generator, but any unique alphanumeric string is fine)
+    ```yaml
+    # secrets.yaml
+    # (...)
+    # Token and Secret Key : from `Developer Option` in the SwitchBot App (version ≥6.14)
+    switchbot_token: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+    switchbot_sec: xxxxxxxxxxxx
+    # Random Value: you can use a UUID generator, but any unique alphanumeric string is OK
+    switchbot_nonc: xxxxx-xxxxx-xxxxx-xxxxx-xxxxx
+    ```
+6. **Link the files in the `pyscript` directory**
+   ```sh
+   # use `mkdir -p pyscript/apps/` if the directory doesn't exist
+   cd pyscript/apps/
+   
+   # Create a symbolic link to the apps directory named switchbot
+   ln -s ../../deps/SwitchBot-API-Script-Caller/pyscript/apps switchbot
+   ```
+   
+### Further Update
+By following this procedure, the script can then be updated with newer version using git.
+```sh
+cd deps/SwitchBot-API-Script-Caller
+git pull
+```
+
+### Installation Notes
+- In order to see the `Developper options` in the Switchbot app, click repetively on the version number in the App's settings
```

Comment:
add minumum version of app as v.6.14 [here](https://github.com/OpenWonderLabs/SwitchBotAPI#open-token-and-secret-key)

Reply:
```suggestion
- In order to see the `Developper options` in the Switchbot app (version ≥6.14), click repetively on the version number in the App's settings
```